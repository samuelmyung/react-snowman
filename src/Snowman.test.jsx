import { it, expect } from "vitest";

import { render, fireEvent } from "@testing-library/react";
import Snowman from "./Snowman";
// import TEST_IMAGES from "./_testCommon.js";

// it('renders without crashing', function () {
//   render(
//     <Snowman
//       photos={TEST_IMAGES}
//       title="images for testing"
//     />
//   );
// });


// it('matches snapshot', function () {
//   const { container } = render(
//     <Snowman
//       photos={TEST_IMAGES}
//       title="images for testing"
//     />
//   );

//   expect(container).toMatchSnapshot();
// });

it('no more than 6 wrong guesses', function () {
    const { container } = render(
        <Snowman
          maxWrong={2}
          images={[]}
          words={["apple"]}
        />
    );

    const button = container.querySelector(".bi-arrow-right-circle");
    fireEvent.click(rightArrow)

    expect(container)
})