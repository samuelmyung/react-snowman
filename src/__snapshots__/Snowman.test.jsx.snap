// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="2"
    />
    <p
      class="Snowman-n-wrong"
    >
      Number wrong: 
      2
    </p>
    <p>
      apple
    </p>
    <p>
      You Lose!
    </p>
    <button
      class="Snowman-restart"
    >
      Restart
    </button>
  </div>
</div>
`;
